diff -u linux-4.4.orig/debian/config/defines linux-4.4/debian/config/defines
--- linux-4.4.orig/debian/config/defines   2016-01-06 18:25:20.000000000 -0500
+++ linux-4.4/debian/config/defines       2016-01-27 10:31:06.450188964 -0500
@@ -29,10 +29,9 @@
 compiler: gcc-4.9
 featuresets:
  none
- rt
 
 [featureset-rt_base]
-enabled: true
+enabled: false
 
 [description]
 part-long-up: This kernel is not suitable for SMP (multi-processor,
